sdk/include/opentelemetry/sdk/trace/tracer.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "opentelemetry/trace/tracer.h"
+
+namespace opentelemetry
+{
+namespace sdk
+{
+namespace trace
+{
+class Tracer : public opentelemetry::trace::Tracer
+{
+public:
+  explicit Tracer(nostd::string_view name, nostd::string_view version);
```

Comment:
Tracer should be taking some sort of TracerConfig argument in its constructor. 

A name and version string won't be enough. Take a look at the javascript design:

https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-tracing/src/BasicTracerRegistry.ts